packages/fuels-code-gen/src/program_bindings/abigen/logs.rs

```diff
@@ -51,3 +66,34 @@ fn generate_log_id_log_formatter_pairs(resolved_logs: &[ResolvedLog]) -> Vec<Tok
         })
         .collect()
 }
+
+pub(crate) fn generate_id_error_codes_pairs(
+    error_codes: impl IntoIterator<Item = (u64, fuel_abi_types::abi::program::ErrorDetails)>,
+) -> Vec<TokenStream> {
```

Comment:
Nit: I think we can return an iterator here, to allow the caller to decide if they want to collect the results or apply any other transformation on them.

Probably overkill here though.

Reply:
Thanks for updating! :pray: